### PR TITLE
Move to VPP master

### PR DESCRIPTION
**Pull request by AloysAugustin**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -473,19 +473,25 @@ func getMaxCIDRMask(addr net.IP) net.IPMask {
 
 func safeAddInterfaceAddress(swIfIndex uint32, addr *net.IPNet) (err error) {
 	maskSize, _ := addr.Mask.Size()
-	if vpplink.IsIP6(addr.IP) && maskSize != 128 {
-		err = vpp.AddInterfaceAddress(swIfIndex, &net.IPNet{
-			IP:   addr.IP,
-			Mask: getMaxCIDRMask(addr.IP),
-		})
+	if vpplink.IsIP6(addr.IP) {
+		err = vpp.EnableInterfaceIP6(swIfIndex)
```
Comment (sknat) [inline main.go]: Ach, yep forgot this. But https://github.com/calico-vpp/vpp-manager/pull/14/files#diff-7ddfb3e035b42cd70649cc33393fe32cR649 should fix it better nope ?

File: main.go [hunk 2]
```diff
@@ -613,7 +619,7 @@ func configureVpp() (err error) {
 			Gw:        route.Gw,
 		})
 		if err != nil {
-			return errors.Wrap(err, "cannot add route in vpp")
+			log.Errorf("cannot add route in vpp: %v", err)
```
Comment (sknat) [inline main.go:616]: Never sure if failing / continuing is better... 
I think here for debugging sake it's better to not fail too fast.

**Review by sknat — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (AloysAugustin) [reply main.go]: Yep, removed this hunk from this PR.

File: main.go (hunk 2 above)
Comment (AloysAugustin) [reply main.go:616]: Right, for now it's either that, or we need to be more careful not to add a route that will already be set with VPP, or we need vpplink to not fail when adding a route that already exists. This is the fastest way forward for now...

File: main.go (hunk 1 above)
Comment (sknat) [reply main.go]: Cool thx 👍

**Review by sknat — APPROVED** [review]

**Pull request merged**